code/game/objects/structures/crates_lockers/closets/secure/engineering.dm

```diff
@@ -39,9 +39,7 @@
 
 /obj/structure/closet/secure_closet/engineering_electrical/WillContain()
 	return list(
-		/obj/item/clothing/gloves/nabber = 2,
-		/obj/item/clothing/gloves/vox = 2,
-		/obj/item/clothing/gloves/insulated = 3,
+		/obj/item/clothing/gloves/insulated = 4,
```

Comment:
id bitch and moan about this not being clearly communicated but keep that as something in mind for future PR's since its minimal issue

Reply:
you should propably still though add this to the discription/PR name for the sake of record keeping

Reply:
look at how ugly the title is now look what you've DONE